.dockerignore

```diff
@@ -0,0 +1,19 @@
+# Files to ignore when building Docker image
+
+# Git data directory
+.git
+
+# Committed files that are not needed in the build
+docker-compose.yml
+README.md
+LICENSE
+
+# Uncommitted files that may by lying around
+*.sql
```

Comment:
I'd take this one out, it only covers `.sql` files but it would be just as easy to have a `.txt` lying around (eg `notes.txt`, `todo.txt`) or other filetype, and trying to guard against uncommitted files could just bite us later when they turn out to be files that we wanted to include